server/app/views/NorthStarBaseView.java

```diff
@@ -158,6 +176,15 @@ protected ThymeleafModule.PlayThymeleafContext createThymeleafContext(
     return context;
   }
 
+  private String getBaseUrlWithoutScheme() {
```

Comment:
```suggestion
  /**
  * Gets the site hostname from the `BASE_URL`.
  * <p>This value does not include the scheme. Do not use this for links.
  */
  private String getSiteHostname() {
```

(blocking) Please add javadoc and update the method name.